sql/2021/security/tls_versions_pages.sql

```diff
@@ -0,0 +1,25 @@
+#standardSQL
+# Distribution of TLS versions on all TLS-enabled web pages
+SELECT
+  client,
+  tls_version,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total_pages,
+  COUNT(0) AS freq,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
+FROM (
+  SELECT
+    client,
+    IFNULL(JSON_EXTRACT_SCALAR(payload, '$._tls_version'), JSON_EXTRACT_SCALAR(payload, '$._securityDetails.protocol')) AS tls_version
+  FROM
+    `httparchive.almanac.requests`
+  WHERE
+    date = '2021-08-01' AND
+    STARTS_WITH(url, 'https') AND
+    firsthtml = true)
```

Comment:
It's actually the `true` it's complaining about (should be `TRUE`) - though to be honest you don't even need that as this works (and is what is used in the other queries):

```suggestion
    firstHtml)
```